<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeRemoteDialog</class>
 <widget class="QDialog" name="ChangeRemoteDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>110</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="remoteURLLabel">
       <property name="text">
        <string>Remote &amp;URL:</string>
       </property>
       <property name="buddy">
        <cstring>remoteURLLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="remoteURLLineEdit">
       <property name="placeholderText">
        <string>wss://example.com/conflip/</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="serverSecretCheckBox">
       <property name="text">
        <string>Server &amp;Secret:</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="serverSecretLineEdit">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>remoteURLLineEdit</tabstop>
  <tabstop>serverSecretCheckBox</tabstop>
  <tabstop>serverSecretLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ChangeRemoteDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ChangeRemoteDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>serverSecretCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>serverSecretLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
